src/tasks/get-category-task.ts

```diff
@@ -1,35 +1,26 @@
 // global
-// TOCHANGE import { CategoryNotFound } from '../../../util/graasp-error';
 import { DatabaseTransactionHandler, Member } from 'graasp'
 // local
 import { CategoryService } from '../db-service';
 import { BaseCategoryTask } from './base-category-task';
 import { Category } from '../interfaces/category';
 
-type InputType = { categoryId?: string };
-
 export class GetCategoryTask extends BaseCategoryTask<Category> {
+  protected categoryId: number;
   get name(): string {
     return GetCategoryTask.name;
   }
 
-  input: InputType;
-  getInput: () => InputType;
-
-  constructor(member: Member, categoryService: CategoryService, input?: InputType) {
+  constructor(member: Member, categoryId: string, categoryService: CategoryService) {
     super(member, categoryService);
-    this.input = input ?? {};
```

Comment:
You shouldn't get rid of input. This is used in the runner to apply dynamic parmeters. We can discuss so I give you more information about it!